public/scripts/wiki.js

```diff
@@ -131,15 +234,18 @@ var ArticleList = React.createClass({
     var parentThis = this;
     var articleTitles = this.props.articles.map(function(article, index){
       return (
-        <li key={index} onClick={parentThis.handleClick.bind(this, article._id)}> {article.title} </li>
+        <div className="article-list-item" key={index} onClick={parentThis.handleClick.bind(this, article._id)}> {article.title} </div>
       );
     });
 
     return (
       <div className="article-list">
-        <ul>  
-          {articleTitles}
-        </ul>
+        <h3><center>Users</center></h3>
+        <div className="article-list-item">Bobby McFingerson</div>
+        <div className="article-list-item">True dads TM</div>
```

Comment:
oops, forgot to get rid of that. I added that in case we wanted to show new users and to also see my changes without having to wait for the page to update. if you think it's not too much to add, then go for it!